CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 cookbook-logstash CHANGELOG
 ===============
 
+## 2.4.1
+
+  - Pablo Pérez
+    - [39bfe8b] lint
```

Comment:
Lines of lint an changes in syntax can be safely discarded from the changelog since they are not significant changes for the task